src/components/common/post/section/child/list/ListPostSection.jsx

```diff
@@ -0,0 +1,20 @@
+import s from "./ListPostSection.module.css"
+import PropTypes from "prop-types";
+
+function ListPostSection({list}) {
+    return (
+        <ul>
+            {
+                list?.map(li =>
+                    <li className={s.post_section_list_child} key={li} dangerouslySetInnerHTML={{__html: li}}/>
```

Comment:
key should always be unique, as long as the li is unique - this is ok.

Reply:
And what if it is not unique? 
Reason I am asking is that I really do not want to declare a lot of ids, in those json files with content I created.